src/test/regress/expected/single_hash_repartition_join.out

```diff
@@ -510,6 +510,21 @@ SELECT count(*) FROM test_numeric t1 JOIN test_numeric as t2 ON (t1.a = t2.b);
   1001
 (1 row)
 
+SET citus.shard_replication_factor to 2;
+CREATE TABLE dist_1 (a int , b int);
+SELECT create_distributed_table('dist_1', 'a');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+SET citus.task_executor_type to 'task-tracker';
+SELECT COUNT(*) FROM dist_1 f, dist_1 s WHERE f.a = s.b;
+ count
+---------------------------------------------------------------------
+     0
```

Comment:
I wonder if we should populate the table with some records first.
Something like the following should do the trick I guess.

```
INSERT INTO dist_1 SELECT x,10-x FROM generate_series(1,10) x
```